src/pages/user/wallet.tsx

```diff
@@ -217,14 +218,18 @@ export default function UserWallet() {
                     </Title>
                     <Group className={classes.lifetimeBuzzBadge} spacing={2}>
                       <CurrencyIcon currency={Currency.BUZZ} size={24} />
-                      <Text
-                        size="xl"
-                        style={{ fontSize: 32, fontWeight: 700, lineHeight: '24px' }}
-                        color="yellow.7"
-                        className={classes.lifetimeBuzz}
-                      >
-                        {numberWithCommas(lifetimeBalance ?? 0)}
-                      </Text>
+                      {lifetimeBalance === null ? (
+                        <Loader variant="dots" />
```

Comment:
Let's match the same color from the text
```suggestion
                        <Loader variant="dots" color="yellow.7" />
```